nx-X11/programs/Xserver/Xext/xf86bigfont.c

```diff
@@ -638,12 +638,12 @@ ProcXF86BigfontQueryFont(
 	        + (nCharInfos+1)/2 * 2 * sizeof(CARD16)
 	      : 0);
 	xXF86BigfontQueryFontReply* reply =
-	   (xXF86BigfontQueryFontReply *) ALLOCATE_LOCAL(rlength);
+	   (xXF86BigfontQueryFontReply *) calloc(1, rlength);
```

Comment:
Why do you use calloc() here? Uninitialized values issue? If so, I guess, there are more malloc()s in this PR that should possibly become calloc()s?